utxo.go

```diff
@@ -0,0 +1,9 @@
+package bt
+
+// UTXO an unspent transaction output, used for creating inputs
+type UTXO struct {
+	TxID          string
+	Vout          uint32
+	LockingScript string
```

Comment:
hmm you think this should be a *bscript.Script like the inputs and outputs? also the TxID be a byte array? https://github.com/libsv/go-bt/blob/master/input.go#L31